polytope/polytope.py

```diff
@@ -442,6 +468,134 @@ def text(self, txt, ax=None, color='black'):
         _plot_text(self, txt, ax, color)
 
 
+def _translate(polyreg, d):
+    """Translate a polyreg by a vector in place. Does not return a copy.
```

Comment:
To clarify my intent, I am citing both forms to indicate that I don't know which one is more widely used. Wikipedia [uses](https://en.wikipedia.org/wiki/In-place_algorithm) the hyphenated version. Your [comment](https://github.com/tulip-control/polytope/pull/39#discussion_r99249122) suggests that "in-place" may be a better choice.